lib/stat_tracker.rb

```diff
@@ -138,4 +147,95 @@ def percentage_visitor_wins
     end
     (games_won_by_visitor.count.to_f / games.count * 100).round(2)
   end
+
+  def count_of_teams
+    @teams.count
+  end
+
+  def games_by_all_team_ids
```

Comment:
This looks good! Like this helper method and I think we could use it for other stuff